src/Common/OpacityTable.cpp

```diff
@@ -286,7 +289,7 @@ FLOAT OpacityTable<ndim>::GetMuBar
 
 //=================================================================================================
 //  OpacityTable::GetGamma()
-/// GetGamma returns gamma for index of density and u
+/// GetGamma returns ratio of specific heats for index of density and u
```

Comment:
> GetGamma returns ratio of specific heats for index of density and u

This is not actually true. Can you just say this is the gamma from P = (gamm-1)rho u?